src/components/ChatLog.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+
+
+
+const ChatLog = (props) => {
+    const entries = props.entries;
+    
+    return (
+        <div className= 'chat-log'>
+        {
+            entries.map((entry,id) => (
+                <ChatEntry 
+                id={entry.id}
+                sender={entry.sender}
+                body={entry.body}
+                timeStamp={entry.timeStamp}
+                liked={entry.liked}
+                updateLikes={props.updateLikes}
+                key={id}
+            />
+            )
+            )}
+        </div>
+    )
+};
+
+
```

Comment:
There's no `PropTypes` set for this component, which is probably fine as it would be redundant with the ones you do set in `ChatEntry`. At the same time, it's usually a good habit to use tools like `PropTypes` to check your expectations, because bugs can crop up in surprising places...